receiver/oracledbreceiver/config.go

```diff
@@ -17,15 +17,24 @@ import (
 )
 
 var (
-	errBadDataSource = errors.New("datasource is invalid")
-	errBadEndpoint   = errors.New("endpoint must be specified as host:port")
-	errBadPort       = errors.New("invalid port in endpoint")
-	errEmptyEndpoint = errors.New("endpoint must be specified")
-	errEmptyPassword = errors.New("password must be set")
-	errEmptyService  = errors.New("service must be specified")
-	errEmptyUsername = errors.New("username must be set")
+	errBadDataSource       = errors.New("datasource is invalid")
+	errBadEndpoint         = errors.New("endpoint must be specified as host:port")
+	errBadPort             = errors.New("invalid port in endpoint")
+	errEmptyEndpoint       = errors.New("endpoint must be specified")
+	errEmptyPassword       = errors.New("password must be set")
+	errEmptyService        = errors.New("service must be specified")
+	errEmptyUsername       = errors.New("username must be set")
+	errMaxQuerySampleCount = errors.New("`max_query_sample_count` must be between 1 and 10000")
+	errTopQueryCount       = errors.New("`top_query_count` must be between 1 and 200 and less than or equal to `max_query_sample_count`")
+	errQueryCacheSize      = errors.New("`query_cache_size` must be strictly positive")
 )
 
+type TopQueryCollection struct {
+	MaxQuerySampleCount uint `mapstructure:"max_query_sample_count"`
+	TopQueryCount       uint `mapstructure:"top_query_count"`
+	QueryCacheSize      int  `mapstructure:"query_cache_size"`
```

Comment:
Why do we need a separate configuration for this? Can we use MaxQuerySampleCount value instead?